<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LconWindow</class>
 <widget class="QMainWindow" name="LconWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lab_pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>571</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);
background-color: rgb(145, 145, 145);</string>
    </property>
    <property name="text">
     <string>图像显示区域</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>571</width>
      <height>58</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里汉仪智能黑体</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_Start">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 16pt;
border: 2px;
border-color:rgb(0, 0, 0);
font: 75 14pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>开始播放</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Stop">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 16pt;
border: 2px;
border-color:rgb(0, 0, 0);
font: 75 14pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>停止播放</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 16pt;
border: 2px;
border-color:rgb(0, 0, 0);
font: 75 14pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>返回连接</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_out">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 16pt;
border: 2px;
border-color:rgb(0, 0, 0);
font: 75 14pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>退出程序</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
